docs/overview/component/index-manager.md

```diff
@@ -0,0 +1,43 @@
+# Vald Index Manager
+
+Vald Index Manager is the component that controls the indexing process of Vald Agent Pods in the same Vald cluster.
+
+## Responsibility
+
+Vald Index Manager has a unique and simple role in preventing stop-the-world.
```

Comment:
Is the role of index manager is to prevent stop-the-world?

It think the main responsibility is to control the timing of indexing, so to prevent the stop-the-world due to NGT's defect.

Reply:
yes, it's right.
I'll update.